scripts/resultsSum.pl

```diff
@@ -148,7 +148,8 @@ sub resultReporter {
 						$endTime = $1;
 						$tapString .= "    duration_ms: " . ($endTime - $startTime) . "\n  ...\n";
 						last;
-					} elsif ($result =~  /(Test results: .*)(passed|skipped|failed|error)(: .*\d{1,}$)/) {
```

Comment:
This works as console output. as `Test results: passed: 4; failed: 6; skipped: 3,912` does show in the TEST TARGETS SUMMARY part. So the easiest way is to update \d{1,}$)/ to .*\d{1,}$)/ in line 204 and 218, which will match coma part.

Reply:
yeah I didn't like the way the ".*" was being used to match the ",", I think that was by accident. The group is giving the "illusion" it is capturing the "number", when it is not...
Hence I took the decision to simplify it and remove the "iilusion", as there is no need to capture the number here, and I think my suggestion simplifies the understanding of what it is doing?
It's illusions like this that imo lead to misstakes.

Thoughts?

Reply:
i'm happy either way.... i just prefer simplicity!

Reply:
Also the reason line :218 incorrectly contains "skipped" is so that it matches, because skipped is the last keyword, so previously the following will match as failed:
Test results: passed: 4; skipped: 3
but removing skipped from the old version would not then match the following:
Test results: passed: 4; failed: 6; skipped: 3

Reply:
Agree, since no need to capture the number here this way is simpler.